en/docs/chapter_graph/graph.md

```diff
@@ -19,18 +19,18 @@ If vertices are viewed as nodes and edges as references (pointers) connecting th
 Based on whether edges have direction, graphs can be divided into <u>undirected graphs</u> and <u>directed graphs</u>, as shown in the figure below.
 
 - In undirected graphs, edges represent a "bidirectional" connection between two vertices, for example, the "friendship" in WeChat or QQ.
-- In directed graphs, edges have directionality, that is, the edges $A \rightarrow B$ and $A \leftarrow B$ are independent of each other, for example, the "follow" and "be followed" relationship on Weibo or TikTok.
+- In directed graphs, edges have directionality, that is, the edges $A \rightarrow B$ and $A \leftarrow B$ are independent of each other, for example, the "follow" and "followed" relationship on Instagram or TikTok.
 
 ![Directed and undirected graphs](graph.assets/directed_graph.png)
 
 Based on whether all vertices are connected, graphs can be divided into <u>connected graphs</u> and <u>disconnected graphs</u>, as shown in the figure below.
```

Comment:
"Depending on" might be better than "Based on"

Reply:
Taken